src/App.js

```diff
@@ -1,7 +1,7 @@
 import "./App.css";
```

Comment:
UI appreciation: Nice that you show a toast at the bottom for login. Nice touch.